migrations/4_deploy_account_ingress_rules_contract.js

```diff
@@ -12,10 +12,16 @@ const rulesContractName = Web3Utils.utf8ToHex("rules");
 let accountIngress = process.env.ACCOUNT_INGRESS_CONTRACT_ADDRESS;
 let retainCurrentRulesContract = AllowlistUtils.getRetainAccountRulesContract();
 
+async function logCurrentAllowlist(instance) {
+    let currentAllowlist = await instance.getAccounts();
+    console.log("\n<<< current ACCOUNT allowlist >>>");
+    console.log(currentAllowlist);
+}
```

Comment:
add log for "end of current ACCOUNT allowlist"

Reply:
done